ui/src/routes/_layout/organizations/$orgId.index.tsx

```diff
@@ -86,6 +86,7 @@ const OrganizationComponent = () => {
         queryFn: async () =>
           await ProductsService.getOrganizationProducts({
             organizationId: Number.parseInt(params.orgId),
+            limit: 1000,
```

Comment:
Actually this should be adjusted, or more specifically the limit should be added [here](https://github.com/eclipse-apoapsis/ort-server/blob/0158f02c36ff15b66865393418d843b5ce1b10e4/ui/src/routes/_layout/organizations/%24orgId.index.tsx#L251-L258) as well, and the same is true for other pages. This way [loader](https://tanstack.com/router/v1/docs/framework/react/guide/external-data-loading#using-loaders-to-ensure-data-is-loaded) uses the same query as the query inside the component.

Not related to this PR, but this should be refactored to use `queryOptions` as shown [here](https://tanstack.com/router/v1/docs/framework/react/guide/external-data-loading#a-more-realistic-example-using-tanstack-query) to not require repetition of the query options.